packages/core-data/src/fetch/index.js

```diff
@@ -1,2 +1,93 @@
-export { default as __experimentalFetchLinkSuggestions } from './__experimental-fetch-link-suggestions';
+/**
+ * WordPress dependencies
+ */
+import { useCallback } from '@wordpress/element';
+import { useSelect } from '@wordpress/data';
+import { store as blockEditorStore } from '@wordpress/block-editor';
+
+/**
+ * Internal dependencies
+ */
+import { store as coreStore } from '../';
+import { default as fetchLinkSuggestions } from './__experimental-fetch-link-suggestions';
+
+export const __experimentalFetchLinkSuggestions = fetchLinkSuggestions;
 export { default as __experimentalFetchUrlData } from './__experimental-fetch-url-data';
+
+export function __experimentalUseLinkControlEntitySearch() {
```

Comment:
Can this be a private API instead of a prefixed one.

Reply:
I am working on a plan to move this fetching to become a full selector in Core Data in order to take advantage of caching...etc. So making it private would be good.

Reply:
See https://github.com/WordPress/gutenberg/pull/57582